Classes/IPNavBarSqueezableViewController.m

```diff
@@ -8,36 +8,51 @@
 
 #import "IPNavBarSqueezableViewController.h"
 
-#define DEBUG_SQUEEZE NO
-#define SCREEN_WIDTH ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) || ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) ? [[UIScreen mainScreen] bounds].size.width : [[UIScreen mainScreen] bounds].size.height)
+/**
+ Uncomment to debug
+ */
+// #define DEBUG_SQUEEZE
 
-#define NAVBAR_HEIGHT ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) || ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) ? 44 : 32)
+#define SCREEN_WIDTH ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) \
+|| ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) \
+? [[UIScreen mainScreen] bounds].size.width : [[UIScreen mainScreen] bounds].size.height)
 
-#define TOOLBAR_HEIGHT ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) || ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) ? 49 : 32)
+#define NAVBAR_HEIGHT ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) \
+|| ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) ? 44.f : 32.f)
 
-#define SQUEEZED_NAVBAR_HEIGHT 20
-#define STATUSBAR_HEIGHT 20
+#define TOOLBAR_HEIGHT ((([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortrait) \
+|| ([UIApplication sharedApplication].statusBarOrientation == UIInterfaceOrientationPortraitUpsideDown)) ? 49.f : 32.f)
+
+
+static const CGFloat kSqueezedNavigationBarHeight = 20.f;
+static const CGFloat kStatusBarHeight             = 20.f;
+static const NSTimeInterval kAnimationDuration    = 0.25;
 
-@interface IPNavBarSqueezableViewController () <UIScrollViewDelegate>
 
 typedef NS_ENUM(NSInteger, IPNavBarSqueezingStatus) {
     IPNavBarSqueezingStatusNormal,
+    IPNavBarSqueezingStatusProgress,
     IPNavBarSqueezingStatusSqueezing,
```

Comment:
Turns out `IPNavBarSqueezingStatusSqueezing` has been replaced by `IPNavBarSqueezingStatusProgress`. Maybe we should removed this status?